infra/base-images/base-builder-swift/Dockerfile

```diff
@@ -14,6 +14,6 @@
 #
 ################################################################################
 
-FROM gcr.io/oss-fuzz-base/base-builder:xenial
+FROM gcr.io/oss-fuzz-base/base-builder-new:xenial
```

Comment:
Can you add a FIXME: that need to move to base-builder and 20.04. need to remember this too. or ask @catenacyber for help.

Reply:
I plan to do that as part of https://github.com/google/oss-fuzz/pull/6312 after this PR is merged and the dust settled

Reply:
cc @jonathanmetzman @inferno-chromium I got swift to work with new ubuntu in #6312
Tested with
```
python3 infra/helper.py build_image --no-pull base-builder
python3 infra/helper.py build_image --no-pull base-builder-swift
python3 infra/helper.py build_image --no-pull swift-protobuf
python3 infra/helper.py build_fuzzers swift-protobuf
```